deepchem/models/keras_model.py

```diff
@@ -146,7 +151,7 @@ def __init__(self,
     else:
       self.optimizer = optimizer
     self.tensorboard = tensorboard
-    self.tensorboard_log_frequency = tensorboard_log_frequency
+    self.log_frequency = log_frequency
```

Comment:
For backward compatibility, should we also check if `'tensorboard_log_frequency'` is in `kwargs`?

Reply:
Good call! I've changed the statement so it's backwards compatible and emits a warning if you use `tensorboard_log_frequency`.